src/layouts/Default.vue

```diff
@@ -35,7 +35,7 @@
                         <h3>Get in touch</h3>
                         <p>
                             <i class="fab fa-gitter"></i>
-                            Discuss with us on <a href="#">Matrix</a>
+                            Discuss with us on <a href="https://matrix.to/#/#gmn:matrix.org">Matrix</a>
```

Comment:
Maybe better a link to the Lobby channel

Reply:
Thanks, done!